### PR TITLE
feat: add ident ring for login profile picture

### PR DESCRIPTION
**Pull request by yqrashawn**
partial fix for subissue 1 in #15788

Adds ident ring for user set profile picture in views before login
ident ring around initials is not supported yet. Will work on that next.

CleanShot 2023-05-18 at 16 37 12
CleanShot 2023-05-18 at 16 37 07

related status-go PR https://github.com/status-im/status-go/pull/3500 https://github.com/status-im/status-go/pull/3505
with these status-go changes
1. media server gets started before login
1. colorhash and colorid are always available in multiaccount db (db avaliable before login)

status: ready

### CODE REVIEW

File: src/utils/image_server.cljs
Comment (yqrashawn) [inline src/utils/image_server.cljs:29]: the `ring?` here will be used later to disable ident ring for user with ens name

File: src/status_im2/subs/onboarding.cljs
Comment (yqrashawn) [inline src/status_im2/subs/onboarding.cljs:61]: image name stored in status-go is the `:type` of images, `"large"`, `"thumbnail"`

**Comment by pavloburykh** [comment]
Hey @yqrashawn ! Thanx for the PR. We are waiting for the review. After approval we will be able to take it into testing.

**Review by alwx — APPROVED** [review]

File: src/status_im2/subs/onboarding.cljs
Comment (flexsurfer) [inline src/status_im2/subs/onboarding.cljs:63]: im not sure about using effect in subscription, can we dispatch`get-account-image-uri` and store result in app-db ?

**Comment by pavloburykh** [comment]
@yqrashawn thank you for the PR. 

Just to make sure: empty profile image is not in scope of current PR, correct? 

photo_2023-05-22 15 26 12

**Comment by pavloburykh** [comment]
> Just to make sure: empty profile image is not in scope of current PR, correct?

If it is not - then this PR is ready to be merged. 

If it is in scope - then we need to add ident ring for empty login profile picture.

**Comment by yqrashawn** [comment]
@pavloburykh it's not in the scope of this PR.
Avatar with no custom pic requires generating images with initials from the status-go side, still WIP https://github.com/status-im/status-go/pull/3513

**Pull request merged**
